test/ch/aggregation_test.exs

```diff
@@ -25,8 +25,8 @@ defmodule Ch.AggregationTest do
       ticker LowCardinality(String),
       time DateTime CODEC(Delta, Default),
```

Comment:
should this also be `DateTime('UTC')`?

Reply:
yeah good catch

Reply:
sorry, but the result row would need to be updates to `~U[]`...

Reply:
yeah just did, sorry my bad...

Reply:
test types date32 (Ch.ConnectionTest)

Also fails for me in addition to the others u mentioned